tests/hf/test_integrals.py

```diff
@@ -282,6 +285,163 @@ def test_gradient(symbols, geometry, alpha, coeff):
     assert np.allclose(g_coeff, g_ref_coeff)
 
 
+@pytest.mark.parametrize(
+    ("alpha", "beta", "t", "e", "rc", "ref"),
+    [
+        (  # trivial case, ref = 0.0 for t > e
+            np.array([3.42525091]),
+            np.array([3.42525091]),
+            2,
+            1,
+            np.array([1.5]),
+            np.array([0.0]),
+        ),
+        (  # trivial case, ref = 0.0 for e == 0 and t != 0
+            np.array([3.42525091]),
+            np.array([3.42525091]),
+            1,
+            0,
+            np.array([1.5]),
+            np.array([0.0]),
```

Comment:
actually this test would trigger the first branch `t > e`, replacing 1 --> -1 addresses this. It's a trivial test, but should still be good to hit the correct branch
```suggestion
        (  # trivial case, ref = 0.0 for e == 0 and t != 0
            np.array([3.42525091]),
            np.array([3.42525091]),
            -1,
            0,
            np.array([1.5]),
            np.array([0.0]),
```